data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/export/ExportScheduler.java

```diff
@@ -95,6 +97,10 @@ public void run() {
         executor.shutdownNow();
     }
 
+    public void shutDown() {
```

Comment:
Please name as `shutdown()` to match other similar methods.

Reply:
Updated.